relationships.md

```diff
@@ -346,6 +346,30 @@ This can be further filtered to a single DSO using
 https://dspace7-entities.atmire.com/rest/#https://dspace7-entities.atmire.com/rest/api/core/relationships/search/byLabel?label=isPersonOfOrgUnit&dso=f2235aa6-6fe7-4174-a690-598b72dd8e44 which contains all relationships created using the relationship type isPersonOfOrgUnit for which one item is f2235aa6-6fe7-4174-a690-598b72dd8e44
 The dso parameter is optional
 
+## Property-based projections
+
+[Property-based projections](projections.md#property-based-projections) can add new JSON properties to the response. When requesting the projection, any `relationship` in the response will add these properties.  
+The REST request doesn't have to be on the `relationship` endpoint directly, it can be on another endpoint which simply embeds relationships.
+
+### Verify whether a given item is the left or right
+**?projection=CheckSideItemInRelationship&checkSideItemInRelationship=<:item-uuid>**
```

Comment:
The name of this projection might be simplified to something like `CheckRelatedItemSide` or `CheckSideOfRelatedItem`.  That's a bit more similar to the projection name off the Items endpoint.

Reply:
I agree that simplifies the name if you know in advance this is specific to a Relationship.
But projections can be applied even if the relationship is embedded. This implies that if you'd be requesting an item, embedding the relationships, and asking to add a projection for the side of the item in the embedded relationships, the name may become a bit vague.
Do you agree to keep the name, or do you expect the name you suggested will be clear enough to indicate it adds the information to a Relationship?